docs/elements/hx-tabcontent/index.html

```diff
@@ -0,0 +1,18 @@
+---
+title: <hx-tabcontent>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="attributes">Attributes</h2>
+  <dl>
+    <dt>
+      role
```

Comment:
Should `role` be in a `<code>...</code>` wrapper?

Reply:
yes good catch, talked to @CITguy and yeah if it displays a value that can be copied and pasted directly into consumer's application code, we should wrap it